test.js

```diff
@@ -142,6 +142,16 @@ test('make bigger swarm', { timeout: 120000 }, async function (t) {
   t.is(firewalled, false)
   t.is(port, swarm[490].address().port)
   t.ok(host)
+
+  // Sanity check: nobody timed out, so there should be no down hints sent
```

Comment:
This assertion isn't directly related to the test, but it made for an easy failing test with the previous behaviour (it sent a few 100 down-hint messages). Can remove the assertion, or make a separate test if you prefer